### PR TITLE
[editor] disable link-preview in inner-rce (until we support atomic plugins in inner-rce)

### PR DESCRIPTION
**Pull request by maimonav**
Currently, we allow only text plugins in inner-rce, but the config of the inner-rce is given from the outer-rce, which contains atomic plugins. So in order to disable link-preview - we need to delete link preview from inner-rce config.
@noamtwix please take a look as well.

### CODE REVIEW

File: packages/editor/web/src/RichContentEditor/InnerRCE.jsx [hunk 1]
```diff
@@ -6,19 +6,33 @@ import RichContentEditor from './RichContentEditor';
...
     this.state = {
       editorState,
     };
   }
 
+  cleanConfig = config => {
+    const configWithoutAnchor = this.removeAnchorFromLink(config);
+    const configWithoutLinkPreview = this.removeLinkPreview(configWithoutAnchor);
+    return configWithoutLinkPreview;
+  };
```
Comment (noamtwix) [inline packages/editor/web/src/RichContentEditor/InnerRCE.jsx:27]: in my opinion it's a bit more clear
```suggestion
  cleanConfig = config => {
    let clearConfig = config;
    clearConfig = this.removeAnchorFromLink(clearConfig);
    clearConfig = this.removeLinkPreview(clearConfig);
    return clearConfig;
  };
```

**Comment by noamtwix** [comment]
@maimonav did you check how it looks when adding the link preview to the inner-rce plugins?

**Comment by maimonav** [comment]
> @maimonav did you check how it looks when adding the link preview to the inner-rce plugins?

Yes, it works ok. But after talking to @oded-soffrin regarding it, we decided that we don't want to expose atomic plugins yet, only in next phase, so for now, we want to disable it.

File: packages/editor/web/src/RichContentEditor/InnerRCE.jsx (hunk 1 above)
Comment (maimonav) [reply packages/editor/web/src/RichContentEditor/InnerRCE.jsx:27]: Sure.

**Review by shaulgo — APPROVED** [review]

**Pull request merged**